WordPress/wordcamp.org
# added orientation tool plugin

Added a fork of [contributor orientation tool](This plugin is a fork or https://github.com/wceu/contributor-orientation-tool)

Fixes #, See #
- This is not a bug fix


Props <username>, <username>
To all the maintainers of contributor orientation tool


### Screenshots

Added illustrations for each team 

![image](https://user-images.githubusercontent.com/22427070/197120505-3c97e73f-5b77-4465-8a45-008637c99bcb.png)






### How to test the changes in this Pull Request:

1. Install the cd orientation tool plugin
2. Activate it
3. Add [contributor-orientation-tool] shortcode and check the output

P.S This plugin is a fork of [contributor orientation tool](This plugin is a fork or https://github.com/wceu/contributor-orientation-tool). We want to add it to the WC Asia website.
